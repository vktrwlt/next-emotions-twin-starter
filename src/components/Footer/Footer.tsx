import tw, { styled, css, theme } from 'twin.macro';
import { Container } from '@/components/Container';
const Box = styled.div`
  ${tw``}
`;

const Footer = () => {
  return (
    <Box>
      <Container>Footer</Container>
    </Box>
  );
};

export { Footer };
