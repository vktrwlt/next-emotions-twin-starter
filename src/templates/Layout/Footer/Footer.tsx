import tw, { styled, css, theme } from 'twin.macro';
import { Container } from '@/components/Container';
const Box = styled.div`
  ${tw``}
`;

export const Footer = () => {
  return (
    <Box>
      <Container>Footer</Container>
    </Box>
  );
};
