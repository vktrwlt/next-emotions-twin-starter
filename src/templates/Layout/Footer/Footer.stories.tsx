import React from 'react';
import { Footer } from '@/templates/Layout/Footer/Footer';

export default {
  title: 'Footer',
  component: Footer,
};

const Template = (args) => <Footer {...args} />;

export const Default = Template.bind({});
Default.args = {};
