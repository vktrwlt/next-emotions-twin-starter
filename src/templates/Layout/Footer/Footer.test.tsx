import { Footer } from '@/templates/Layout/Footer/Footer';
import { render, fireEvent } from '@/utils/test-utils';

describe('<Footer />', () => {
  const { asFragment } = render(<Footer />);
  it('should render without crashing', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
