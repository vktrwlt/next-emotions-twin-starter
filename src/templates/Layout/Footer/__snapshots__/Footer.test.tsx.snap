// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render without crashing 1`] = `
<DocumentFragment>
  .emotion-2 {
  max-width: 1280px;
  padding-left: 2rem;
  padding-right: 2rem;
  margin-left: auto;
  margin-right: auto;
}

<div
    class="emotion-0 emotion-1"
  >
    <div
      class="emotion-2 emotion-3"
    >
      Footer
    </div>
  </div>
</DocumentFragment>
`;
